src/export.js

```diff
@@ -61,3 +61,10 @@ if ( typeof module !== "undefined" && module && module.exports ) {
 if ( typeof exports !== "undefined" && exports ) {
 	exports.QUnit = QUnit;
 }
+
+if ( typeof define === "function" && define.amd ) {
+	define( [], function() {
+		return QUnit;
+	} );
```

Comment:
> we can remove the [] argument. Ref http://www.requirejs.org/docs/api.html#deffunc

ok

> Also, I'm not up to date on the docs, but it shouldn't close as }); instead of } );?

Not anymore: http://contribute.jquery.org/style-guide/js/#spacing